api/validator_test.go

```diff
@@ -34,6 +34,24 @@ var _ = Describe("Validator", func() {
 		errList = ValidateDisruptionSpecFromString(yamlDisruptionSpec.String())
 	})
 
+	Describe("validating disruption triggers", func() {
+		Context("both offset and notBefore are set", func() {
+			BeforeEach(func() {
+				yamlDisruptionSpec.WriteString("\nnetwork:")
```

Comment:
for readability you might want to use HEREDOC style strings, just be careful to NOT let your autoformatter re-indent the string for you.:

```suggestion
yamlDisruptionSpec.WriteString(`
network:
  corrupt: 100
  duration: 87600h
  triggers
    pods:
      notBefore: 2040-01-02T15:04:05-04:00
      offset: 1m
`)
```

Reply:
I'm having a lot of trouble with this approach ^, going to leave it as is

Reply:
Okay, all I had to do was give up on editing this file in GoLand and do it in vim.